pkg/apis/internal.acorn.io/v1/unmarshal.go

```diff
@@ -257,13 +257,15 @@ func impliedVolumesForContainer(app *AppSpec, containerName, sideCarName string,
 					// ignore error
 					continue
 				}
-				vSize, err := resource.ParseQuantity((string)(v.Size))
-				if err != nil {
-					// ignore error
-					continue
-				}
-				if existingSize.Cmp(vSize) < 0 {
-					existing.Size = v.Size
+				if v.Size != "" {
+					vSize, err := resource.ParseQuantity((string)(v.Size))
+					if err != nil {
+						// ignore error
+						continue
+					}
```

Comment:
Worth logging? When would this happen?

Reply:
As far as I can tell, this is not really possible. Additionally, this isn't being changed.